plugins/chat/app/services/chat/create_direct_message_channel.rb

```diff
@@ -42,6 +42,7 @@ class Contract
       attribute :name, :string
       attribute :target_usernames, :array
       attribute :target_groups, :array
+      attribute :chat_link, :boolean, default: false
```

Comment:
It's not very clear to me what chat_link is going to do just from reading the name

Reply:
Maybe `upsert` true/false, would be a better option name?